src/components/ChatBotMessages/index.jsx

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import ChatBot from "react-simple-chatbot";
+import { ThemeProvider } from "styled-components";
+import { useTranslation } from "react-i18next";
+import womanBot from "./images/woman.svg";
+import womanSeeksHelp from "./images/womanSeeksHelp.svg";
+import "./index.scss";
+import { Redirect } from "react-router-dom";
+
+const CHATBOT_THEME = {
+  background: "#FFFEFC",
+  fontFamily: "Roboto",
+  headerBgColor: "#FFBFB5",
+  headerFontColor: "#fff",
+  headerFontSize: "15px",
+  botBubbleColor: "#C8D7C2",
+  botFontColor: "#fff",
+  userBubbleColor: "#FFBFB5",
+  userFontColor: "#fff",
+};
+
+const BotRedirectMessage = ({ path, message }) => {
```

Comment:
prefer to make as few things know about translation as possible. also if you wanted this to be a translation key the param should be called translationKey, not message.

but its cleaner for components to be "stupid." BotRedirectMessage doesnt know about the concept of translations. it doesnt care. it just takes a string and shows it.

basically the t() should be on line 98, not in this component.

Reply:
yes, you are right, i forgot to delete it